examples/js/loaders/OBJLoader.js

```diff
@@ -589,7 +587,9 @@ THREE.OBJLoader = ( function () {
 
 					state.materialLibraries.push( line.substring( 7 ).trim() );
 
-				} else if ( ( result = smoothing_pattern.exec( line ) ) !== null ) {
+				} else if ( lineFirstChar === "s" ) {
+
+					result = line.length > 1 ? line.substring( 1 ).trim().split( " " ) : [];
```

Comment:
Yes, thanks, that's better. The more complicated statement was a left over from detection by regex.